### DISCUSSION TITLE
New version 1.3.0 not working anymore on intel Mac

### DISCUSSION THREAD

**Discussion by Jampsu76** [latenitefilms/BRAWToolbox#146] [Support] [answered]
Plugin is not working anymore. How can I downgrade it?
Everything was fine before the latest update. Can't view my clips anymore...

I have a 2020 intel based iMac 27 with Blackmagic eGPU Pro.

**Comment by Jampsu76** [comment]
Edit. The plugin works but not with an earlier clips. So I must do everything again.

**Reply by latenitefilms** [reply]
Ummm... that's very odd. Did you update from **v1.2.2 (54)** to **v1.3.0 (55)**?

The only change we made in the last update was to the Blackmagic RAW SDK, which should still work on both Intel and Apple Silicon. If importing your files still works, then that's unrelated.

Can you share a screenshot of what's not working exactly after updating?

**Reply by jpmeia** [reply]
I had the same problem.
--BRAW files imported and successfully used with the "two-prior"version to 1.3.0(55), then not used
--Tried to access after upgrading to 1.3.0 (55),a a variety of problems:
     --"Plugin not responding, plugin disabled" alert panel
     --Another icon I get when BRAW fails to work is:

“Missing BRAW File.  This BRAW Toolbox clip is currently offline.  This means either the security-scoped bookmark or file path to the BRAW file is invalid.  If you haven’t moved the file, and the path is th same, you can try. “Grant Sandbox Access” to the drive containing your BRAW clip.  If the file has been moved, you can use the “Relink BRAW Clipts with an LIBRARY/EVENT/PROJECT” Toolbox

At first, using the 1.3.0(55) Buttonbar for "relinking" resulted in the same problems.  I ended up finding an "Uninstall BRAW toolbox" uninstaller on GitHub, probably for an older version.  Then deleted the existing BRAW Toolbox "directory" in Applications folder, downloading anew from App Store, creating a new Library in FCPX 10.6.6, and using the BRAW from scratch.  So far everything works.

Here are some screenshots of my system, FCPX, and BRAW Toolbox Alert Panels:
![Image](https://github.com/latenitefilms/BRAWToolbox/assets/139928755/9bdce073-19cd-40b3-a461-c7e45c4b176e)

![Image](https://github.com/latenitefilms/BRAWToolbox/assets/139928755/e982596e-8b9a-4147-b640-52d821fefef8)
![Image](https://github.com/latenitefilms/BRAWToolbox/assets/139928755/c7a2b8d7-5843-4ced-8b9b-b1b9a6ef70d0)

![image](https://github.com/latenitefilms/BRAWToolbox/assets/139928755/3940acf1-5a05-437a-a3d1-094f7de7e4c1)
![Image](https://github.com/latenitefilms/BRAWToolbox/assets/139928755/9b9a9e59-26e9-42d8-bd8f-07e7ca22c876)
![Image](https://github.com/latenitefilms/BRAWToolbox/assets/139928755/2dd6a849-480e-4163-9e4e-8bf718ffde4b)

I am new to GitHub (created the account to comment here), sorry if this comment is disjoint.  Three things I can say:
==BRAW Toolbox 1.3.0(55) ButtonBar for "relinking" seen in the screenshot above DOES NOT WORK
==For my Intel Mac system running latest Monetary (see screenshot), BRAW Toolbox only works if I start a new Library within FCPX10.6.6 and use BRAW Toolbox to import from scratch.
==For at least two users, simply upgrading to 1.3.0(55) breaks existing workflow between BRAW toolbox and FCPX.

Hope this helps, I also sent an email to latenitefilms with similar info.  Please let me know if I can help further. 

Still thanks for the BRAW Toolbox, we would be dead in the water without it!

**Reply by latenitefilms** [reply]
Sorry to hear you're both running into issues...

> I ended up finding an "Uninstall BRAW toolbox" uninstaller on GitHub, probably for an older version. 

I think the problem you're running into is that when you uninstalled BRAW Toolbox, is basically breaks all the security scoped bookmarks because you deleted the `~/Library/Group Containers/A5HDJTY9X5.com.latenitefilms.BRAWToolbox/Library/Application Support/BRAWToolbox.document` file. 

You can work around this by just granting sandbox access to the drive that contains all your BRAW footage as explained here:

https://brawtoolbox.io/collaboration/

> BRAW Toolbox 1.3.0(55) ButtonBar for "relinking" seen in the screenshot above DOES NOT WORK

Are you following the instructions below?

https://brawtoolbox.io/toolbox/#relink-braw-clips-within-an-library--event--project

It's working fine here?

> For my Intel Mac system running latest Monetary (see screenshot), BRAW Toolbox only works if I start a new Library within FCPX10.6.6 and use BRAW Toolbox to import from scratch.

This shouldn't be the case. It's possible it's a bug, or it could be a misunderstanding of what happens when you uninstalled BRAW Toolbox.

> For at least two users, simply upgrading to 1.3.0(55) breaks existing workflow between BRAW toolbox and FCPX.

Are you able to email me your log files as explained here:

https://brawtoolbox.io/troubleshooting/#ive-run-into-a-bug-where-can-i-find-the-log-files

**Comment by jpmeia** [comment]
HI Chris,

Thanks for your very quick reply.

So far, my “fix” of deleting the Library created with the two-prior version of BRAW-Toolbox, and creating a new Library with newly-imported-via-BRAWToolbox-files has held up, even after updating just now to FCPX 10.6.7

re the “Uninstaller”, that was a last-ditch effort at the very end of my trying things, so I don’t think that did anything at all.  On our backup laptop (similar to mine, slightly better CPU) I simply deleted the original library, created a new Library, and used 1.3.0(55) BRAW-Toolbox to import from scratch, and that worked fine without any uninstall/re-installs.  That as well is working fine so far.

For sure for me, on two nearly-identical MBP-intel laptops, re-linking using BRAW-Toolbox did not work.

Will get the log files you want and send them shortly.

thanks again Chris, really appreciate your quick follow-up and **most excellent** work with BRAW toolbox.  

Later, JP

**Comment by jpmeia** [comment]
Hi Chris,

Here is a .zip of the requested files.  If these aren’t very revealing of a problem, I can send a set from our other laptop, which did exhibit
the same problems, although on that laptop I did not do any unistall/re-installs, just experienced the problems and started from scratch with a new
Library and BRAW-Toolbox imports.

Here is a question I haven’t found a good answer to—what are the best BRAW-Toolbox Buttonbar options to use for importing BRAW files
for use on Macs and FCPX without any need for Resolve?  Do these choices matter?  All I am doing is importing BRAWs into FCPX, so not even
sure if the “Toolbox” Buttonbar settings matter for that; thanks for any insight.

After reading your email, it seems I did NOT follow the proper procedure for re-linking within BRAW-Toolbox, so any of my commentary there is invalid.

Thanks again, JP

**Reply by latenitefilms** [reply]
Unfortunately your ZIP file hasn't come through on GitHub.

> what are the best BRAW-Toolbox Buttonbar options to use for importing BRAW files for use on Macs and FCPX without any need for Resolve?

I'm not sure I quite understand your question? Do you mean, what are the best settings in the BRAW Toolbox Workflow Extension? Most users generally just use the RAW controls to set the desired Color Space/Gamut and Gamma, and correct any mistakes, such as incorrect Color Temperature or Exposure, then just do the actual grading in Final Cut Pro. Of course, you can always change the RAW data later if you need to - that's the benefit of RAW - nothing is locked in.

**Comment by Jampsu76** [comment, marked as answer]
Thanks for the reply and help, both of you guys. 
I had exactly same problem than JP did. Thanks for the screenshots. 
I wasn't able to take screenshots anymore, because I deleted my old files (all my braw toolbox clips was just a test clips for now on) and now the plugin seems to work ok.

I have tho some issues with the workflow with the plugin, but maybe I make another topic for that.

**Reply by latenitefilms** [reply]
Glad to hear everything is now working. Feel free to create another discussion if you still have workflow questions - happy to help!

**Comment by jpmeia** [comment]
Zip file—I can try to re-send this, although I don’t think it will help as I think the problems I (and probably JampSu76) had was simply User Error--did not relink files using the procedure in brawtoolbox.io/toolbox.  When I tried using the “Relink BRAW clips….” selection in the “Toolbox” area, I did not Drag anything on to the Toolbox area, just listed the files I wanted to relink into the File List area, set the Toolbox Buttonbar to “Relink BRAW clips….”, and tried a new import process.  So I never got the “Relink BRAW Files” panel with the Status lights.

Which leads to my question about the “Workflow Extension” and now I can better phase it:  Do the settings in the “Toolbox” portion of the Workflow Extension matter if I am simply importing BRAW files via at the File List area of the Workflow Extension, and not dragging any Libraries/Events onto the “Toolbox” area at the bottom of the Workflow extension?

If you still want the Zip file I can send it along, and if there is some test you need me to try let me know and I will do that.  Appreciate your help and of course, the BRAW workflow!

Thanks Chris, JP
